packages/models/organizations.coffee

```diff
@@ -7,8 +7,13 @@ Organization = Astro.Class
     name: 'string'
     description: 'string'
     createdById: 'string'
-
   methods:
+    addMember: (userId) ->
+      UserProfiles.update({userId: userId}, {
+        $addToSet: {memberOfOrgs: @_id}
+      })
```

Comment:
Not necessarily: https://forums.meteor.com/t/meteor-methods-vs-allow-deny/1010/5

And I think using server-side methods will be more compatible with astronomy.